samples/nrf9160/aws_iot/README.rst

```diff
@@ -167,6 +169,20 @@ When the sample runs, the device boots, and the sample displays the following ou
         } to AWS IoT broker
         Next data publication in 60 seconds
 
+To observe incoming messages in the AWS IoT console, navigate to the AWS IoT Core service and
+click on :guilabel:`MQTT test client`.
+To observe messages that are sent from the device, subscribe to the **$aws/things/<client_id>/shadow/update** shadow topic or the wild card token **#**.
+
+.. figure:: /images/aws_mqtt_test_client.png
```

Comment:
```suggestion
.. figure:: /images/aws_mqtt_test_client.PNG
```
Either change the file extension here to uppercase or change it in the image file to lowercase.